src/main/java/com/gruppe27/fellesprosjekt/client/controllers/CreateEventController.java

```diff
@@ -191,10 +195,28 @@ private void handleComboBoxClicked() {
         //TODO Validering
     }
 
+    LocalTime toLocalTime(String time) {
+        try {
```

Comment:
Byttet ut denne med den over. Stian tiitet på de andre DateTimeFormatterne og han så ingen som støttet det direkte. Kanskje det er mulig hvis man implementerer interfacet selv, men vi vil uansett måtte ha en "toLocalTime"-metode for å enforce at alle bruker samme parser.